File: java/get_campaign_status.java
import okhttp3.*;

import java.io.IOException;

public class Main {
    public static void main(String[] args) throws IOException {
        OkHttpClient client = new OkHttpClient().newBuilder().build();

        MediaType mediaType = MediaType.parse("text/plain");
        RequestBody body = RequestBody.create(mediaType, "");

        Request request = new Request.Builder()
                .url("<REPLACE_WITH_PROD_URL>/v1/campaigns/1/status")
                .method("GET", body)
                .addHeader("Authorization", "Bearer <YOUR_TOKEN>")
                .build();

        Response response = client.newCall(request).execute();
        String responseBody = response.body().string();
        System.out.println(responseBody);
    }
}

File: php/get_campaign_status.php
<?php
// GUZZLE
// Make sure to install Guzzle using Composer: composer require guzzlehttp/guzzle

use GuzzleHttp\Client;
use GuzzleHttp\Psr7\Request;

// Replace the following values with your own
$prodUrl = '<REPLACE_WITH_PROD_URL>';
$token = '<YOUR_TOKEN>';

$client = new Client();
$headers = [
    'Authorization' => 'Bearer ' . $token
];
$body = '';
$request = new Request('GET', $prodUrl . '/v1/campaigns/1/status', $headers, $body);
$res = $client->sendAsync($request)->wait();
echo $res->getBody();

